### PR TITLE
replace action

### PR DESCRIPTION
**Pull request by chirag04**
absolutely untested code. 😆 

@skevy thoughts on whats the best way to test local changes? 

PS: I now know you feel strongly about shipping transpiled code to npm.

### CODE REVIEW

File: src/ExNavigationStack.js [hunk 1]
```diff
@@ -112,6 +112,14 @@ export class ExNavigationStackContext extends ExNavigatorContext {
     });
   }
 
+  @debounce(500, true)
+  replace() {
```
Comment (chirag04) [inline src/ExNavigationStack.js]: route param missing here

**Comment by chirag04** [comment]
ok. ninja tested it by adding this PR to transpiled code under node_modules 😆

**Comment by chirag04** [comment]
navbar does not work well tho.

**Comment by chirag04** [comment]
ok fixed the navbar problems.

**Comment by chirag04** [comment]
fixes https://github.com/exponentjs/ex-navigation/issues/13

File: src/ExNavigationStack.js [hunk 2]
```diff
@@ -462,13 +472,17 @@ class ExNavigationStack extends PureComponent<any, Props, State> {
   };
 
   _renderScene = (props: ExNavigationSceneRendererProps) => {
+    // filter out stale scenes
+    const scenes = props.scenes.filter(scene => !scene.isStale);
```
Comment (skevy) [inline src/ExNavigationStack.js]: 👍  I wouldn't be surprised if this fixes other weird bugs.

**Comment by skevy** [comment]
LGTM. Thanks for doing this @chirag04! Maybe `replaceAtIndex` is next? :-p

**Comment by chirag04** [comment]
can do `replaceAtIndex` but i want to clean up node-haste and transpilation stuff.

**Pull request closed without merging**